bus/client.go

```diff
@@ -105,6 +105,12 @@ func (c *Client) BroadcastTransaction(ctx context.Context, txns []types.Transact
 	return c.c.WithContext(ctx).POST("/txpool/broadcast", txns, nil)
 }
 
+// WalletInfo calls the /wallet endpoint on the bus.
+func (c *Client) WalletInfo(ctx context.Context) (resp api.WalletResponse, err error) {
```

Comment:
should we rename it to just `Wallet`? seems like the most consistent thing to do, I was kind of expecting the route to be `/wallet/info` because of it